Mathlib/Algebra/Module/LinearMap/Basic.lean

```diff
@@ -21,7 +21,7 @@ open Function
 
 universe u u' v w x y z
 
-variable {R R₁ R₂ R₃ k S S₃ T M M₁ M₂ M₃ N₁ N₂ N₃ ι : Type*}
+variable {R R₂ S M M₂ : Type*}
```

Comment:
Can you rename R2 to e.g. R' or R1? Right now, this looks a bit funky...

Reply:
I used `R R' M M'`.